### ISSUE TITLE
Switching Votes is cumbersome

### ISSUE THREAD

**Issue by Jarunik** [ArkEcosystem/desktop-wallet#86]
It is too much effort to switch your vote from one delegate to another.

Currently:
1) Unvote
2) Wait 
3) Vote 

It is not clear how long you have to wait. It would be good to see somehow when the unvote is done and you can vote again.

It would be even better if you could unvote and vote someone else at the same time (like the voting list indicates but does not let you do it).

**Comment by alexbarnsley** [comment]
@fix @Jarunik is this still a problem? I can see you can add a delegate and vote, which unvoted the other.

**Comment by boldninja** [comment]
You cannot vote and unvote in same tx.

**Comment by alexbarnsley** [comment]
I know, I realised when I tried it 😬 I've got an idea on how to make it easier when doing both at the same time, without having to do separate actions. It will still involve separate transactions but I think I'll do a wait on the unvote (with the option of cancelling on a dialog), so it will wait until the unvote transaction is confirmed, then perform the new vote. It makes sense in my head. I'll implement for people to check/test.
